v0/sources/satismeter/transform.js

```diff
@@ -0,0 +1,28 @@
+const Message = require("../message");
+const { refinePayload } = require("../../util");
+const trackMapping = require("./data/trackMapping.json");
+
+const buildTrackPayload = event => {
+  const message = new Message(`SATISMETER`);
+  message.setEventType("track");
+  message.setPropertiesV2(event, trackMapping);
+  message.event = `Survey ${event.event}`;
+  return message;
+};
+
+const processEvent = event => {
+  const message = buildTrackPayload(event);
+  // making sure we are returning either userId or AnonymousId
+  if (!message?.userId && !message?.anonymousId) {
+    message.anonymousId = event?.response.id; // fallback to response.id as response.id is always present in payload
+  }
+  return message;
+};
+
+const process = event => {
+  const response = processEvent(event);
+  const returnValue = refinePayload(response);
```

Comment:
Can we have a more informative name here in place of return value, something related to the response or payload.

Reply:
Done